src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/bulkscan/transformer/D8FormToCaseTransformer.java

```diff
@@ -0,0 +1,33 @@
+package uk.gov.hmcts.reform.finrem.caseorchestration.service.bulkscan.transformer;
+
+import uk.gov.hmcts.reform.finrem.caseorchestration.model.bulkscan.OcrDataField;
+import uk.gov.hmcts.reform.finrem.caseorchestration.model.bulkscan.transformation.in.ExceptionRecord;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class D8FormToCaseTransformer implements ExceptionRecordToCaseTransformer {
+
+    private static Map<String, String> d8ExceptionRecordToCcdMap() {
+        Map<String, String> map = new HashMap<>();
+        map.put("firstName", "D8FirstName");
+        map.put("lastName", "D8LastName");
+
+        return map;
+    }
+
+    @Override
+    public Map<String, Object> toCase(ExceptionRecord exceptionRecord) {
+        Map<String, Object> resultingCaseData = new HashMap<>();
+
+        List<OcrDataField> allOcrKeyValuePairs = exceptionRecord.getOcrDataFields();
+        allOcrKeyValuePairs.forEach((erEntry) ->
```

Comment:
You can use collect(Collections.toMap()).
Have a look into this: https://github.com/hmcts/div-case-orchestration-service/pull/595/files#diff-d191dfbe31223ba66f4b7c85d4c7f29cR25

Reply:
Great idea, much more concise!